src/main/scala/io/iohk/ethereum/vm/OpCode.scala

```diff
@@ -656,6 +656,70 @@ case object LOG3 extends LogOp(0xa3)
 case object LOG4 extends LogOp(0xa4)
 
 
+case object CREATE extends OpCode(0xf0, 3, 1, G_create) {
+  protected def exec[W <: WorldStateProxy[W, S], S <: Storage[S]](state: ProgramState[W, S]): ProgramState[W, S] = {
+    val (Seq(endowment, inOffset, inSize), stack1) = state.stack.pop(3)
+
+    val validCall = state.env.callDepth < OpCode.MaxCallDepth && endowment <= state.ownBalance
+
+    if (!validCall) {
+      val stack2 = stack1.push(UInt256.Zero)
+      state.withStack(stack2).step()
+    } else {
+
+      val (initCode, memory1) = state.memory.load(inOffset, inSize)
+      val (newAddress, world1) = state.world.newAddress(state.env.ownerAddr)
+      val world2 = world1.transfer(state.env.ownerAddr, newAddress, endowment)
+
+      val newEnv = ExecEnv(
+        newAddress,
+        state.env.callerAddr,
```

Comment:
You're on 🔥 
Yes, it should be `ownerAddr` and it's good idea to add those symbols to the comments.